src/frontend/src/index.tsx

```diff
@@ -1,18 +1,31 @@
 import * as React from 'react';
-import { registerApp, ContextTypes, Context } from '@equinor/fusion';
+import { registerApp, ContextTypes, Context, useFusionContext } from '@equinor/fusion';
 import { Switch, Route } from 'react-router-dom';
 import LandingPage from './pages/LandingPage';
-// import ContractsOverviewPage from './pages/ContractsOverviewPage';
-// import ContractDetailsPage from './pages/ContractDetailsPage';
 import ProjectPage from './pages/ProjectPage';
+import ApiClient from './api/ApiClient';
+import AppContext from './appContext';
 
 const App: React.FC = () => {
+    const fusionContext = useFusionContext();
+    const apiClient = new ApiClient(
```

Comment:
Memorize client?

Reply:
Shouldn't be that many rerenders of the app component, but sure we can memoize the client

Reply:
Don't bother